data/src/main/java/ch/srg/dataProvider/integrationlayer/data/remote/SRGMediaMetadata.kt

```diff
@@ -28,12 +27,12 @@ interface SRGMediaMetadata : SRGIdentifierMetadata, SRGImageMetadata, SRGMetadat
     fun getDownloadUri(quality: Quality = Quality.HD): String? {
         return when (quality) {
             Quality.SD -> podcastSdUrl
-            Quality.HD, Quality.HQ -> if (TextUtils.isEmpty(podcastHdUrl)) podcastSdUrl else podcastHdUrl
+            Quality.HD, Quality.HQ -> if (podcastHdUrl.isNullOrEmpty()) podcastSdUrl else podcastHdUrl
```

Comment:
Or maybe even isNullOrBlank ?

Reply:
Done in https://github.com/SRGSSR/srgdataprovider-android/pull/46/commits/365c40cc79d80d7144d1ce5546180cf045ee3c57